routes/web.php

```diff
@@ -29,6 +29,13 @@
     Route::get('about', 'AboutController@index');
     Route::get('authlog', 'UserController@authlog');
 
+    // Maps
+    Route::group(['prefix' => 'maps', 'namespace' => 'Maps'], function () {
+        Route::get('devicedependency', 'DeviceDependencyController@dependencyMap');
+        Route::get('devicedependency/group={group_id}', 'DeviceDependencyController@dependencyMap');
+        Route::get('devicedependency/group={group_id}/highlight_node={highlight_node}', 'DeviceDependencyController@dependencyMap');
```

Comment:
This should be removed and use `$request->get('highlight_node')` instead

Reply:
@Jellyfrog 
tried to remove both lines with variables
and in `DeviceDependencyController` changed 

```
public function dependencyMap(Request $request, $group_id = 0, $highlight_node = 0)
{
```
to
```
    public function dependencyMap(Request $request)
    {
    $group_id = $request->get('group_id') | 0;
    $highlight_node = $request->get('highlight_node') | 0;
```
but with this everytime GET arguments are in URL Page fall's back to Main Page

some Ideas?

Reply:
@Jellyfrog 
Code changed